src/main/scala/util/Replacement.scala

```diff
@@ -301,6 +302,17 @@ class SetAssocLRU(n_sets: Int, n_ways: Int, policy: String) extends SetAssocRepl
     state_vec(set) := logic.get_next_state(state_vec(set), touch_way)
   }
 
+  def access(sets: Seq[UInt], touch_ways: Seq[Valid[UInt]]) = {
+    require(sets.size == touch_ways.size)
```

Comment:
Do you think you could add in the second argument to `require()`, which is the assertion message? It's pretty painful to debug the error messages and stack traces coming from these things as a person who doesn't actually work on the Chisel code.

Reply:
If it’s an internal consistency check, I don’t think the message is necessary—but if a particular set of configuration parameters could cause the assertion to fire, I totally agree.

Reply:
it's an internal consistency check

Reply:
I still think it's a good idea to err on the side of adding messages, even for "internal consistency checks". Whether or not something "internal" or "externally configurable" may change over time, and the person who makes that change may not know about all the individual `require()` statements that were internal.

Anyway, thanks @ingallsj for adding a message!

Reply:
Fair enough